### PR TITLE
Global changes

### PR DESCRIPTION
**Pull request by alajurczak**
I created the basis of the application from which we will start. I also added theme and globalStyle.

### CODE REVIEW

File: src/theme.js
Comment (PawelNackowski) [inline src/theme.js:1]: I think those are good names.

Comment (PawelNackowski) [reply src/theme.js:1]: Color whisper shuld not be "#E5E5E5"  ?

**Review by AllmostHumann — CHANGES_REQUESTED** [review]
Hey, instead adding normalize with "<link>" I suggest to edit App.js file and import Normalize from styled-normalize. Then just add code <Normalize> under <GlobalStyle> :D

**Review by AllmostHumann — COMMENTED** [review]
Looks perfect!

**Pull request merged**